### PR TITLE
Te user profile

### PR DESCRIPTION
**Pull request by Testell**

### CODE REVIEW

**Review by jessjno — APPROVED** [review]
looks good to me!

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully created." }
+        format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
```
Comment (marywebby) [inline app/controllers/comments_controller.rb:29]: Good job implementing the redirect_back here!

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by!(username: params.fetch(:username))
```
Comment (marywebby) [inline app/controllers/users_controller.rb:3]: yay!

**Review by marywebby — APPROVED** [review]
Good job! Everything looks good to me

File: app/views/users/show.html.erb
Comment (jessjno) [inline app/views/users/show.html.erb:1]: nice formatting!

File: app/views/comments/_form.html.erb
Comment (jessjno) [inline app/views/comments/_form.html.erb:1]: nice job refactoring

File: config/routes.rb
Comment (jessjno) [inline config/routes.rb:1]: looks good!

Comment (dsimp) [reply config/routes.rb:1]: Nice!

**Pull request merged**